logbook-api/src/main/java/org/zalando/logbook/HttpStatus.java

```diff
@@ -0,0 +1,100 @@
+    SEE_OTHER(303, "See Other"),
+    NOT_MODIFIED(304, "Not Modified"),
+    USE_PROXY(305, "Use Proxy"),
+    SWITCH_PROXY(306, "Switch Proxy"),
+    TEMPORARY_REDIRECT(307, "Temporary Redirect"),
+    PERMANENT_REDIRECT(308, "Permanent Redirect"),
+
+    // 4xx
+    BAD_REQUEST(400, "Bad Request"),
+    UNAUTHORIZED(401, "Unauthorized"),
+    PAYMENT_REQUIRED(402, "Payment Required"),
+    FORBIDDEN(403, "Forbidden"),
+    NOT_FOUND(404, "Not Found"),
+    METHOD_NOT_ALLOWED(405, "Method Not Allowed"),
+    NOT_ACCEPTABLE(406, "Not Acceptable"),
+    PROXY_AUTHENTICATION_REQUIRED(407, "Proxy Authentication Required"),
+    REQUEST_TIMEOUT(408, "Request Timeout"),
+    CONFLICT(409, "Conflict"),
+    GONE(410, "Gone"),
+    LENGTH_REQUIRED(411, "Length Required"),
+    PRECONDITION_FAILED(412, "Precondition Failed"),
+    PAYLOAD_TOO_LARGE(413, "Payload Too Large"),
+    URI_TOO_LONG(414, "URI Too Long"),
+    UNSUPPORTED_MEDIA_TYPE(415, "Unsupported Media Type"),
+    REQUESTED_RANGE_NOT_SATISFIABLE(416, "Requested Range Not Satisfiable"),
+    EXPECTATION_FAILED(417, "Expectation Failed"),
+    I_AM_A_TEAPOT(418, "I'm a teapot"),
+    MISDIRECTED_REQUEST(421, "Misdirected Request"),
+    UNPROCESSABLE_ENTITY(422, "Unprocessable Entity"),
+    LOCKED(423, "Locked"),
+    FAILED_DEPENDENCY(424, "Failed Dependency"),
+    UPGRADE_REQUIRED(426, "Upgrade Required"),
+    PRECONDITION_REQUIRED(428, "Precondition Required"),
+    TOO_MANY_REQUESTS(429, "Too Many Requests"),
+    REQUEST_HEADER_FIELDS_TOO_LARGE(431, "Request Header Fields Too Large"),
+    CONNECTION_CLOSED_WITHOUT_RESPONSE(444, "Connection Closed Without Response"),
+    UNAVAILABLE_FOR_LEGAL_REASONS(451, "Unavailable For Legal Reasons"),
+    REQUEST_HEADER_TOO_LARGE(494, "Request header too large"),
+    SSL_CERTIFICATE_ERROR(495, "SSL Certificate Error"),
+    SSL_CERTIFICATE_REQUIRED(496, "SSL Certificate Required"),
+    HTTP_REQUEST_SENT_TO_HTTPS_PORT(497, "HTTP Request Sent to HTTPS Port"),
+    CLIENT_CLOSED_REQUEST(499, "Client Closed Request"),
+
+    // 5xx
+    INTERNAL_SERVER_ERROR(500, "Internal Server Error"),
+    NOT_IMPLEMENTED(501, "Not Implemented"),
+    BAD_GATEWAY(502, "Bad Gateway"),
+    SERVICE_UNAVAILABLE(503, "Service Unavailable"),
+    GATEWAY_TIMEOUT(504, "Gateway Timeout"),
+    HTTP_VERSION_NOT_SUPPORTED(505, "HTTP Version Not Supported"),
+    VARIANT_ALSO_NEGOTIATES(506, "Variant Also Negotiates"),
+    INSUFFICIENT_STORAGE(507, "Insufficient Storage"),
+    LOOP_DETECTED(508, "Loop Detected"),
+    NOT_EXTENDED(510, "Not Extended"),
+    NETWORK_AUTHENTICATION_REQUIRED(511, "Network Authentication Required"),
+    NETWORK_CONNECT_TIMEOUT_ERROR(599, "Network Connect Timeout Error");
+
+    private final int code;
+    private final String reasonPhrase;
+
+    HttpStatus(int code, String reasonPhrase) {
+        this.code = code;
+        this.reasonPhrase = reasonPhrase.intern();
+    }
+
+    public static Map<Integer, String> toMap() {
```

Comment:
As this map is created every time to just make a single lookup, what do you think about having a method that will just return the reason phrase by code? It will look similar, but will not collect the results in a map, and will just apply the filtering directly here. 
It'll help with readability a bit, It's not obvious what the value in the map is, without looking into this file. `HttpStatus.toMap().get(getStatus())` doesn't tell me that I should get the reason phrase, but `HttpStatus.getReasonPhraseByCode(int code)` will.

Reply:
@kasmarian 

 You're absolutely right. I've made the necessary changes.